src/cli.ts

```diff
@@ -10,21 +10,21 @@ const pkg = require('../package');
 /**
  * Run the script with the user-supplied arguments.
  */
-export default function run(args: Array<string>): void {
+export default async function run(args: Array<string>): Promise<void> {
   let options = parseArguments(args);
 
   if (options.paths.length) {
-    runWithPaths(options.paths, options);
+    await runWithPaths(options.paths, options);
   } else {
-    runWithStdio(options);
+    await runWithStdio(options);
   }
 }
 
-type CLIOptions = {
-  paths: Array<string>,
-  baseOptions: Options,
-  modernizeJS: boolean,
-};
+interface CLIOptions {
```

Comment:
Was changing `type` to `interface` necessary here? I had viewed them as pretty much interchangeable, but looks to me like interface is actually preferred if possible (e.g. it gives better error messages), was that the rationale?

Reply:
I've seen hand-wavy things indicating that interface is usually better, including [this tslint rule](https://palantir.github.io/tslint/rules/interface-over-type-literal/), so I figured I'd just change it while I was there.